src/package.json

```
@@ -108,7 +108,8 @@
     "test": "mocha --import=tsx --timeout 120000 --recursive tests/backend/specs ../node_modules/ep_*/static/tests/backend/specs",
     "test-container": "mocha --import=tsx --timeout 5000 tests/container/specs/api",
     "dev": "node --import tsx node/server.ts",
-    "prod": "node --import tsx node/server.ts"
+    "prod": "node --import tsx node/server.ts",
+    "ts-check": "tsc --noEmit --watch"
```

```suggestion
    "ts-check": "tsc --noEmit",
    "ts-check:watch": "tsc --noEmit --watch"
```

it's allow us to use `ts-check` on CI and `ts-check:watch` for dev. What do you think ?